packages/fluentui/react-northstar/src/themes/teams-v2/components/Chat/chatMessageVariables.ts

```diff
@@ -0,0 +1,35 @@
+import { pxToRem } from '../../../../utils';
+import { ChatMessageVariables } from '../../../teams/components/Chat/chatMessageVariables';
+
+export const chatMessageVariables = (siteVars): ChatMessageVariables => ({
+  actionMenuBoxShadow: siteVars.shadowLevel1,
+  actionMenuPositionRight: pxToRem(5),
+  actionMenuPositionTop: pxToRem(-30),
+  backgroundColor: siteVars.colorScheme.default.background,
+  backgroundColorMine: siteVars.colorScheme.brand.background1,
+  borderRadius: pxToRem(3),
+  color: 'rgb(64, 64, 64)',
+  offset: pxToRem(100),
+  padding: pxToRem(16),
+  authorMarginRight: pxToRem(12),
```

Comment:
Variable clean up is needed

Reply:
Sorry I am stupid/new and didn't understand this comment. Do you mean remove unnecessary variables or clean up wrong values in variables ?

Reply:
I would also like to understand more about 'variable cleanup' - do you think there are variables that are not used in styles?

Reply:
`authorMarginRight` is used in styles. I will check the other ones

Reply:
`actionMenuPositionRight` and `actionMenuPositionTop` are not used in styles and can be removed. would you consider removing them a breaking chance @layershifter? they are in the interface

Reply:
clarified offline - changed to Partial<...> and removed unnecessary variable overrides